scss/forms/_form-text.scss

```
@@ -0,0 +1,10 @@
+//
+// Form text
+//
+
+.form-text {
+  display: block;
+  margin-top: $form-text-margin-top;
+  font-size: $form-font-size;
```

We should use the font-size mixin here:
```suggestion
  @include font-size($form-font-size);
```

I thought we disallowed `font-size` properties at some point, gotta check since when we didn't blacklist this property anymore.

---

Oh yeah!
I'm not yet updated, but getting there :D
Just a sec.

---

Done. Also rebased and squash with the relevant commit.